jobrunner/run.py

```diff
@@ -292,6 +292,7 @@ def handle_job_api(job, api):
         # successful state change to the expected next state
         if new_status.state == ExecutorState.PREPARING:
             job.state = State.RUNNING
+            job.started_at = datetime.datetime.now()
```

Comment:
You want `int(time.time())` here

Reply:
Ha! Beat me to it

Reply:
Yes, thanks. Helpfully the tests let me know.